src/platform/suecreek/include/platform/lib/memory.h

```diff
@@ -402,7 +402,9 @@
 #define BOOT_LDR_STACK_SIZE		SOF_STACK_TOTAL_SIZE
 
 /* code loader entry point for base fw */
-#define SRAM_VECBASE_RESET	(BOOT_LDR_BSS_BASE + BOOT_LDR_BSS_SIZE)
+#define _SRAM_VECBASE_RESET (BOOT_LDR_BSS_BASE + BOOT_LDR_BSS_SIZE)
+#define SRAM_VECBASE_RESET ((_SRAM_VECBASE_RESET) - ((_SRAM_VECBASE_RESET) % \
```

Comment:
You could also use ALIGN_UP from `<sof/common.h>`. Your choice though.

Reply:
@paulstelian97 actually he can use only ALIGN from the linker, not ALIGN/ALIGN_UP from common.h :)
and its a good idea because ALIGN was already used in linker code

Reply:
Guess I don't quite like that there is the other form as well in the comment but will still approve. I want a second opinion on whether including that is good or not.

Reply:
Can we use a common macro to do this alignment.